pkg/kubelet/kubelet_node_status.go

```diff
@@ -161,6 +165,35 @@ func (kl *Kubelet) tryRegisterWithAPIServer(node *v1.Node) bool {
 	return false
 }
 
+// syncFundamentalSystemLabelWithExistingNode synchronize the fundamental system label
```

Comment:
What does fundamental system label mean? Were you referring to "standard system labels"?